tests/cpp/helpers.h

```diff
@@ -23,6 +23,8 @@ long GetFileSize(const std::string filename);
 
 std::string CreateSimpleTestData();
 
+std::string CreateBigTestData(size_t n_iters);
```

Comment:
I think this parameter should be called "n_entries"?

Reply:
Done.